doc/AST_FORMAT.md

```diff
@@ -876,6 +889,17 @@ Format:
  ~~~ name
 ~~~
 
+### Optional single block argument
```

Comment:
I think both of these should go just before "Expression arguments" and gain a small note about their difference from `|a,|`.